.github/actions/submit/action.yaml

```
@@ -32,6 +32,33 @@ outputs:
 runs:
   using: composite
   steps:
+    - name: Bypass aproxy (for private-endpoint runners)
+      if: startsWith(runner.name, 'private-')
+      shell: bash
+      run: |
+        echo "::group::Bypass aproxy (private-endpoint runner)"
+        sudo DEBIAN_FRONTEND=noninteractive apt-get -qqy update && \
+        sudo DEBIAN_FRONTEND=noninteractive apt-get install -qqy dnsutils
```

Since these two commands are joined by `&&`, could you indent the second line?

```suggestion
        sudo DEBIAN_FRONTEND=noninteractive apt-get -qqy update && \
          sudo DEBIAN_FRONTEND=noninteractive apt-get install -qqy dnsutils
```

Additionally, we should probably just export `DEBIAN_FRONTEND=noninteractive` at the start of this step; I can't think of a reason for that not to be set throughout anyways.

---

> we should probably just export DEBIAN_FRONTEND=noninteractive at the start of this step; I can't think of a reason for that not to be set throughout anyways.

I could be wrong but I don't think the commands that run after `sudo` inherit the environment of the parent shell. I will try it out.

---

You are right, I think it requires a `-E` flag, but that's a bit of a big security hole.

---

So. `DEBIAN_FRONTEND=noninteractive` for the update was only necessary because I was using the `-y` option. That doesn't really need to be there for the update, so now the line is much shorter.